Core/Graphics/Include/Babylon/Graphics.h

```diff
@@ -16,9 +16,15 @@ namespace Babylon
         template<typename... Ts>
         static std::unique_ptr<Graphics> CreateGraphics(Ts...);
 
+        template<typename... Ts>
+        static std::unique_ptr<Graphics> CreateGraphicsFromContext(Ts...);
```

Comment:
This pattern is going away anyway in @Drigax's change (which this change should probably wait for to use the new pattern), but if we stop using `void*` then this shouldn't be necessary. The point of the template trick was to allow specializations based on the arguments so that if you just passed it typed pointer to a context, it would automatically look for a specialization that would take that context and create `Graphics` from it, without increasing the surface area of the API.